File: app/src/main/java/com/grarak/ytfetcher/utils/server/playlist/Playlist.java
package com.grarak.ytfetcher.utils.server.playlist;

import com.google.gson.annotations.SerializedName;
import com.grarak.ytfetcher.utils.server.Gson;

public class Playlist extends Gson {

    public String apikey;
    public String name;
    @SerializedName("public")
    public boolean isPublic;
}

File: app/src/main/java/com/grarak/ytfetcher/utils/server/playlist/Playlist.kt
package com.grarak.ytfetcher.utils.server.playlist

import com.google.gson.annotations.SerializedName
import com.grarak.ytfetcher.utils.server.Gson

class Playlist : Gson() {

    var apikey: String? = null
    var name: String? = null
    @SerializedName("public")
    var isPublic: Boolean = false
}
